src/Plutarch/Extra/Bool.hs

```diff
@@ -40,3 +41,19 @@ pcompare t1 t2 ifLT ifEQ ifGT =
 -}
 pcond :: forall (s :: S) (a :: S -> Type). [Term s a -> Term s a] -> Term s a -> Term s a
 pcond = appEndo . foldMap Endo
+
+{- | If the condition evaluated to true, return the third argument. Otherwise error
+     out with the error message.
+
+     @since 3.14.1
+-}
+passert ::
```

Comment:
always fascinating to me which functions we were missing. I could have _sworn_ there was a function that did this already, but looking through hoogle it appears there wasn't!

Reply:
There's always something missing. Just added a few as I drafted the pr on agora side: https://github.com/Liqwid-Labs/agora/blob/fc695e46c7fa9bf74ee2b57317188f0e0c407c41/agora/Agora/Utils.hs